diff --git a/node_modules/@rn-primitives/portal/dist/index.js b/node_modules/@rn-primitives/portal/dist/index.js
index f34a15e..76da5ec 100644
--- a/node_modules/@rn-primitives/portal/dist/index.js
+++ b/node_modules/@rn-primitives/portal/dist/index.js
@@ -40,7 +40,7 @@ module.exports = __toCommonJS(src_exports);
 // src/portal.tsx
 var React = __toESM(require("react"));
 var import_react_native = require("react-native");
-var import_zustand = require("zustand");
+var import_zustand = require("zustand/react");
 var DEFAULT_PORTAL_HOST = "INTERNAL_PRIMITIVE_DEFAULT_HOST_NAME";
 var usePortal = (0, import_zustand.create)(() => ({
   map: (/* @__PURE__ */ new Map()).set(DEFAULT_PORTAL_HOST, /* @__PURE__ */ new Map())
diff --git a/node_modules/@rn-primitives/portal/dist/index.mjs b/node_modules/@rn-primitives/portal/dist/index.mjs
index 952d829..fe50cf8 100644
--- a/node_modules/@rn-primitives/portal/dist/index.mjs
+++ b/node_modules/@rn-primitives/portal/dist/index.mjs
@@ -3,7 +3,7 @@
 // src/portal.tsx
 import * as React from "react";
 import { Platform } from "react-native";
-import { create } from "zustand";
+import { create } from "zustand/react";
 var DEFAULT_PORTAL_HOST = "INTERNAL_PRIMITIVE_DEFAULT_HOST_NAME";
 var usePortal = create(() => ({
   map: (/* @__PURE__ */ new Map()).set(DEFAULT_PORTAL_HOST, /* @__PURE__ */ new Map())
